src/test/java/dev/monogon/cue/lang/parser/CueParserTest.java

```diff
@@ -27,7 +31,16 @@ public CueParserTest(@NotNull String filePath) {
 
     @Test
     public void parser() throws IOException {
-        doTest(true, !getTestName().toLowerCase().contains("error"));
+        try {
+            doTest(true, !getTestName().toLowerCase().contains("error"));
```

Comment:
The platform already supports a system property to automatically override the test data on disk:
```
systemProperties["idea.tests.overwrite.data"] = true
```
Please revert this change, I'll add a run configuration to `master` with sets the property